### ISSUE TITLE
cannot fetch with pagination

### ISSUE THREAD

**Issue by mrwonderfulness** [ccxt/ccxt#8464]
- MacOS
- Python
- latest version of ccxt

I'm not sure if it's the script in your examples folder, or the module, but I keep getting the message below when fetching with pagination. The weird thing is that **_SOMETIMES_** it works. Is there a better way to do pagination so as to not ever get an error?

```
<ipython-input-181-ae3d7f2935d0> in scrape_ohlcv(exchange, max_retries, symbol, timeframe, since, limit)
     28         ohlcv = retry_fetch_ohlcv(exchange, max_retries, symbol, timeframe, fetch_since, limit)
     29         # if we have reached the beginning of history
---> 30         if ohlcv[0][0] >= earliest_timestamp:
     31             break
     32         earliest_timestamp = ohlcv[0][0]

TypeError: 'NoneType' object is not subscriptable
```

**Comment by kroitor** [comment]
@mrwonderfulness thx, we will fix the example. Can you share, which exchange it was, or some more details?

**Comment by mrwonderfulness** [comment]
Yes, that was for Binance... the fetch to csv script from the examples folder. Thanks.

**Comment by ttodua** [comment]
This is an old topic, but just to add note:
- there has not been provided any further follow-ups, and by looking whatever was provided, there is some chance that the error might be coming because there are no items in the timerange you requested , or some argument might have been incorrect.  We dont see what is "retry_fetch_ohlcv" method and what it does, so, we might be unable to provide much insights unless we are provided the full path of execution (including the sample code used throughout the execution).
Let us know if you still experience any issue.
